### PR TITLE
Add common commands & package versions

### PR DESCRIPTION
**Pull request by ntrel**
I'm quite new to dub, so this may need expanding for other commands.
I couldn't easily find anything like this.
Is the version syntax documented anywhere?
Also, I'm guessing how to write diet-ng syntax as I can't build the docs (see #49).

### CODE REVIEW

File: views/getting_started.dt [hunk 1]
```diff
@@ -10,6 +10,24 @@ block body
...
+		li #[code dub build] - build the project in the current folder
+		li #[code dub run] - build & run the project in the current folder (default)
+		li #[code dub run package] - build & run the latest version of `package`
+		li #[code dub fetch package] - fetch the latest version of `package`
+
+	p Other versions of `package` can be specified:
+
+	ul
+		li #[code package@1.0] - the 1.0 version of `package`
+		li #[code package@~master] - the latest git `master` version of `package`
```
Comment (WebFreak001) [inline views/getting_started.dt:27]: the branch versioning is considered deprecated - also I don't really think it's that necessary to know either of these two advanced versioning syntaxes in the overview page. The documentation of the commands should help with that.

Also you may be interested in https://docs.webfreak.org, which is GitHub on https://github.com/WebFreak001/dub-docs-v2, which has CLI documentation as well as more extended documentation pages and is all written in markdown.

**Review by WebFreak001 — CHANGES_REQUESTED** [review]

File: views/getting_started.dt (hunk 1 above)
Comment (ntrel) [reply views/getting_started.dt:27]: Oh great, thanks I'll take a look.

Comment (ntrel) [reply views/getting_started.dt:27]: > the branch versioning is considered deprecated

When I use `--version`, dub says that's deprecated:
```sh
$ dub --version
DUB version 1.33.0, built on Jun  2 2023
$ dub fetch diet-ng --version 2.0.0
     Warning The '--version' parameter was deprecated, use diet-ng@2.0.0. Please update your scripts.
```
What should I be using?

> I don't really think it's that necessary to know either of these two advanced versioning syntaxes in the overview page. The documentation of the commands should help with that.

OK, but where is that documented? I can't find it. Occasionally it mentions one, but it doesn't seem to explain it, e.g. does a leading `~` mean git branch?

Comment (ntrel) [reply views/getting_started.dt:27]: OK I have found an issue, it seems it's not documented:
https://github.com/WebFreak001/dub-docs-v2/issues/12

**Pull request closed without merging**